services/user.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+
+
+def create_user(
+        username: str, password: str, email: str = None,
+        first_name: str = None, last_name: str = None
+) -> None:
+    user_model = get_user_model()
+    user = user_model.objects.create_user(username=username, password=password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+
+
+def get_user(user_id: int) -> get_user_model():
```

Comment:
The return type of the `get_user` function should specify the actual user model class, not `get_user_model()`. Consider using the user model directly for type annotations.